modules/cluster-logging-log-forward-update.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+// Might not be needed. See https://issues.redhat.com/browse/LOG-654
+
+[id="cluster-logging-log-forward-update_{context}"]
+= Updating log forwarding custom resources
+
+The {product-title} Log Forward API has been promoted from Technology Preview to Generally Available in {product-title} 4.6. The GA release contains some improvements and enhancements that require changes in your Cluster Logging custom resource (CR) and Log Forwarding custom resource (CR).
+
+If you use the Log Forwarding API to forward logs, after the Elasticsearch Operator and Cluster Logging Operator are fully updated to 4.6, you must make the following modifications:
+
+. Edit the Cluster Logging custom resource (CR) to remove the `logforwardingtechpreview` annotation:
++
+.Sample Cluster Logging CR
+[sourcce,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  annotations:
+    clusterlogging.openshift.io/logforwardingtechpreview: enabled <1>
+  name: "instance"
+  namespace: "openshift-logging"
+....
+
+----
+<1> Remove the remove the `logforwardingtechpreview` annotation.
+
+. Edit the Log Forwarding CR to use some renamed parameters:
++
+.Sample Log Forwarding CR in {product-title} 4.5
+[sourcce,yaml]
+----
+apiVersion: "logging.openshift.io/v1alpha1"
+kind: "LogForwarding"
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  disableDefaultForwarding: true 
+  outputs:
+   - name: elasticsearch
+     type: "elasticsearch"
+     endpoint: https://elasticsearch.openshift-logging.svc:9200 <3>
+  pipelines:
+   - name: mylogs
+      inputSource:
+        - logs.infra
+        - logs.app   
+        - logs.audit
+     outputRefs:
+     - elasticsearch
+----
++
+.Sample updated Log Forwarding CR in {product-title} 4.6
+[sourcce,yaml]
+----
+apiVersion: "logging.openshift.io/v1" <1>
+kind: "ClusterLogForwarder" <2>
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec: <3>
+  outputs:
+   - name: elasticsearch
+     type: "elasticsearch"
+     url: https://elasticsearch.openshift-logging.svc:9200 <4>
+  pipelines:
+   - name: mylogs
+      inputRefs: <5>
+        - infrastructure
+        - application   
+        - audit
+     outputRefs:
```

Comment:
@alanconway thoughts on referencing the default store here?  This sample would otherwise stop routing messages to the default correct?